Subquery Alias causing panic

I tried to do a more efficient distinct query (the second query) and noticed that the generation panics on the alias of a subquery. The first query shows a simpler example on how to cause this issue.

```
panic: walk: unexpected node type *pg_query.Alias
```

```sql
-- schema.sql
CREATE TABLE authors (
  id BIGSERIAL PRIMARY KEY,
  name TEXT NOT NULL,
  bio TEXT
);
```

```sql
-- name: SubAuthors :many
SELECT sub.name FROM (SELECT * FROM authors) sub

-- name: ListDistinctAuthors :many
WITH RECURSIVE temp (t) AS (
    (SELECT name from authors LIMIT 1)
    UNION ALL
    (SELECT name FROM temp, LATERAL (SELECT name FROM authors where name > t LIMIT 1) sub)
)
SELECT t from temp;
```

--- a/internal/dinosql/soup.go
+++ b/internal/dinosql/soup.go
@@ -854,19 +854,25 @@ func Walk(f Visitor, node nodes.Node) {
 
 	case nodes.RangeFunction:
 		walkn(f, n.Functions)
-		walkn(f, n.Alias)
+		if n.Alias != nil {
+			walkn(f, *n.Alias)
+		}
 		walkn(f, n.Coldeflist)
 
 	case nodes.RangeSubselect:
 		walkn(f, n.Subquery)
-		walkn(f, n.Alias)
+		if n.Alias != nil {
+			walkn(f, *n.Alias)
+		}
 
 	case nodes.RangeTableFunc:
 		walkn(f, n.Docexpr)
 		walkn(f, n.Rowexpr)
 		walkn(f, n.Namespaces)
 		walkn(f, n.Columns)
-		walkn(f, n.Alias)
+		if n.Alias != nil {
+			walkn(f, *n.Alias)
+		}
 
 	case nodes.RangeTableFuncCol:
 		if n.TypeName != nil {
@@ -889,7 +895,9 @@ func Walk(f Visitor, node nodes.Node) {
 		walkn(f, n.ValuesLists)
 		walkn(f, n.Coltypes)
 		walkn(f, n.Colcollations)
-		walkn(f, n.Alias)
+		if n.Alias != nil {
+			walkn(f, *n.Alias)
+		}
 		walkn(f, n.Eref)
 		walkn(f, n.SecurityQuals)
 

